src/components/button.js

```diff
@@ -0,0 +1,5 @@
+function button(textContent,htmlClass,cssClass) {
```

Comment:
why css class?

Reply:
for the buttons of the **top** and **bottom**,but i think it's better change their style with `nth-child()` over just the class(i was thinking  about this today)

Reply:
you are receiving 2 parameters for the same thing

Reply:
`cssClass` does not make sense, you don't have any css code into this component, only `html`

Reply:
how the same thing? one class it's for js apply the event and the other it's for css apply the style

Reply:
it's the same thing!

you can pass 

htmlClass = `A`
cssClass = 'B'

or

htmlClass = 'A B'

or

cssClass = `A B`

Reply:
yea that's true :) 
I thought cause it has just one parameter i only could pass one argument, but "two arguments" here it's like one :o

Reply:
> yea that's true :) I thought cause it has just one parameter i only could pass one argument, but "two arguments" here it's like one :o

so fresh